api/v1beta1/common.go

```diff
@@ -136,6 +136,9 @@ type AnsibleEESpec struct {
 	ExtraMounts []storage.VolMounts `json:"extraMounts,omitempty"`
 	// Env is a list containing the environment variables to pass to the pod
 	Env []corev1.EnvVar `json:"env,omitempty"`
+    // +kubebuilder:validation:Optional
+    // Extra vars to be passed to ansible process during execution.
+	ExtraVars map[string]json.RawMessage `json:"extraVars,omitempty"`
```

Comment:
The field needs to be added to OpenStackDataPlaneDeployment in api/v1beta1/openstackdataplanedeployment_types.go and I would call it AnsibleExtraVars

Reply:
this has been done